tests/post_training/data/wc_reference_data.yaml

```
@@ -1,5 +1,7 @@
 tinyllama_data_free_backend_OV:
   metric_value: 0.72057
+  num_int4: 228
+  num_int8: 84
 tinyllama_data_aware_backend_OV:
   metric_value: 0.83084
```

I can't commit to your branch, here are the numbers. The same for the rest of configurations.
```suggestion
  metric_value: 0.83084
  num_int4: 184
  num_int8: 128
```

---

BTW, in your development, you could try to comment some parts of validate function to not evaluate models for accuracy (similarity). Potentially, it could save some memory and make it possible to check all configurations.